lib/gui/components/safe-webview.js

```diff
@@ -46,6 +46,28 @@ const VERSION_PARAM = 'etcher-version';
 const ELECTRON_SESSION = 'persist:success-banner';
 
 /**
+ * @summary API version search parameter key
```

Comment:
Minor nit, but could we hyphenate this as `search-parameter` ? To stop people thinking that "version search" is a thing ;)
And could we update the `@summary` for `ETCHER_VERSION_PARAM` so that the two read similarly? (and given their similarity, perhaps `ETCHER_VERSION_PARAM` and `API_VERSION_PARAM` should be moved next to each other in the source-code?)